articles/relnotes/index.md

```diff
@@ -17,19 +17,33 @@ For installation instructions, please refer to the [install guide](xref:microsof
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update).
 
 
+## Version 0.xx.xxxx.xxxx
+
+*Release date: February 27th, 2020*
+
+This release contains the following:
+
+- New Quantum Machine Learning library, for more information go to our [QML docs page](https://docs.microsoft.com/quantum/libraries/machine-learning/?view=qsharp-preview)
+-  IQ# bug fixes, resulting in a 10-20x performance increase on the Quantum Katas
```

Comment:
The 10×–20× speedup is indeed an IQ# issue, and affects loading of packages; that is, it's not a general speedup. Since the katas have not been updated to the newest version of IQ#, there may not be a change in performance there for this release. The other IQ# bug fix was provided by @crazy4pi314, and fixes little-endian vs big-endian in the enhanced Jupyter-based state display.

Reply:
Per my deleted comment, I'll update the katas once the release is public.

Reply:
@cgranade, is there a suggestion to update the phrasing here, or is your comment just clarifying where the speedup is coming from?

Reply:
My concern is that the speedup is not in general; performance isn't 10× faster for all IQ# operations, just the package loading. I'd suggest clarifying in the release notes so that users aren't disappointed when not _everything_ is faster.

Reply:
@bettinaheim update this to say "up to" so marking as resolved & merging the PR